core/src/main/java/com/scalar/db/transaction/consensuscommit/Coordinator.java

```diff
@@ -119,7 +119,7 @@ private void put(Put put) throws CoordinatorException {
         break;
       } catch (NoMutationException e) {
         logger.warn("mutation seems applied already", e);
-        throw new CoordinatorException("mutation seems applied already.", e);
+        throw new CoordinatorConflictException("mutation seems applied already.", e);
```

Comment:
Changed to throw `CoordinatorConflictException` when catching `NoMutationException` because I think we should handle when caching `NoMutationException` or `ExecutionException` differently.